### PR TITLE
feat(ctao): email service and more

### PR DESCRIPTION
**Pull request by sigruntg**
# Add confirmation email and other

[221vh39](https://app.clickup.com/t/221vh39)
[wkuhfm](https://app.clickup.com/t/wkuhfm)
[221vgap](https://app.clickup.com/t/221vgap)
[221vk62](https://app.clickup.com/t/221vk62)
[221vgv3](https://app.clickup.com/t/221vgv3)

## What

* Add email service.
* Make sure data schema works as intended.
* Fix text in file uploader.
* Add DoesNotRequireAnswer to custom fields and do not require answers.
* Add file component to overview.

## Why

* Added email service to project so the applicant will get confirmation email.
* File component as design.
* Small fixes to make it ready for deployment.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/complaints-to-althingi-ombudsman/src/forms/ComplaintsToAlthingiOmbudsmanSubmitted.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  buildForm,
+  buildSection,
+  Form,
+  FormModes,
+} from '@island.is/application/core'
+import { confirmation } from '../lib/messages'
+
+export const ComplaintsToAlthingiOmbudsmanSubmitted: Form = buildForm({
+  id: 'ComplaintsToAlthingiOmbudsmanSubmitted',
+  title: 'title',
```
Comment (Thorrik58) [inline libs/application/templates/complaints-to-althingi-ombudsman/src/forms/ComplaintsToAlthingiOmbudsmanSubmitted.ts:12]: should this be "" instead?

**Review by Thorrik58 — COMMENTED** [review]
Minor comment

File: libs/application/template-api-modules/src/lib/modules/templates/complaints-to-althingi-ombudsman/complaints-to-althingi-ombudsman.module.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { DynamicModule } from '@nestjs/common'
+
+// This is a shared module that gives you access to common methods
```
[the remark this answers is not in the document]
Comment (sigruntg) [reply libs/application/template-api-modules/src/lib/modules/templates/complaints-to-althingi-ombudsman/complaints-to-althingi-ombudsman.module.ts]: Done

File: libs/application/templates/complaints-to-althingi-ombudsman/src/forms/ComplaintsToAlthingiOmbudsmanSubmitted.ts (hunk 1 above)
Comment (sigruntg) [reply libs/application/templates/complaints-to-althingi-ombudsman/src/forms/ComplaintsToAlthingiOmbudsmanSubmitted.ts]: Done

**Review by eirikurn — APPROVED** [review]

**Review by Thorrik58 — APPROVED** [review]

**Pull request merged**